lib/features/give/pages/giving_page.dart

```diff
@@ -22,7 +22,6 @@ class GivingPage extends StatefulWidget {
   const GivingPage({
```

Comment:
I would also here seperate logic between new family app and US. Create a new givingpage for the family app :).
Also better as soon as we need to reuse the browser for other purposes (for example donation approval)